Cargo.toml

```diff
@@ -52,6 +52,7 @@ futures = "0.3"
 parking_lot = "0.12"
 tokio = { version = "1", features = ["io-util"] }
 rustc-semver = "1.1"
+toml = "0.5"
```

Comment:
NIT: This is not a UI test dep in that sense. So I would group it with the other test dependencies in lines 31-35.